api/tests/python/end_points/test_namex_search.py

```diff
@@ -603,63 +605,53 @@ def test_namex_search_submitted_start_and_end_date_invalid_date_format(client,
         assert 'Must be of date format %Y-%m-%d' in msg
 
 
-def test_namex_search_direct_nrs_bad_roles(client, jwt, app):
-    """Test searching directly using name request numbers with bad roles."""
-    base_nrs = generate_nrs(5, [], [], [])
-    for nr in base_nrs:
-        nr.save_to_db()
-    qs = "&".join(["nrNumbers=" + nr.nrNum for nr in base_nrs])
-    rv = client.get(
-        f'api/v1/requests?{qs}',
-        headers=create_header(jwt, [])
-    )
-    assert rv
-    assert rv.status_code == 403
-
-
-def test_namex_search_direct_nrs(client, jwt, app):
-    """Test searching directly using name requests."""
-    base_nrs = generate_nrs(5, [], [], [])
-    for nr in base_nrs:
-        nr.save_to_db()
-
-    qs = "&".join(["nrNumbers=" + nr.nrNum for nr in base_nrs])
-    rv = client.get(
-        f'api/v1/requests?{qs}',
-        headers=create_header(jwt, [User.APPROVER, User.EDITOR, User.VIEWONLY])
-    )
-
-    assert rv
-    assert rv.status_code
-    assert rv.status_code == 200
-    assert rv.data
-    resp = json.loads(rv.data.decode('utf-8'))
-    assert len(resp) == 5
-
-
 @pytest.mark.parametrize('test_name, identifiers, total_results', [
     ('Search for NRs by identifier', ['NR 0', 'NR 1', 'NR 2', 'NR 3'], 4),
     ('Empty Search', [], 0),
 ])
-def test_request_search(client, jwt, app, test_name, identifiers, total_results):
-    """Test request search end point."""
-    names = [
+def test_namex_search_direct_nrs(
+    client, jwt, app, test_name, identifiers, total_results
+):  # pylint: disable=unused-argument
+    """Test searching directly using name requests."""
+    base_names = [
         [{'name': 'test1', 'state': 'NE', 'choice': 1}],
         [{'name': 'test 1', 'state': 'NE', 'choice': 1}],
         [{'name': 'test tester 1', 'state': 'NE', 'choice': 1}],
         [{'name': 'testing tester 1', 'state': 'NE', 'choice': 1}],
         [{'name': 'test tester 1', 'state': 'NE', 'choice': 1}]
     ]
-    generate_nrs(5, [], names, [])
+    base_applicants = [
+        create_applicant('1', 'ted', email_address='ted', phone_number='1'),
```

Comment:
can you add this to the generate_nrs fn with a default set? That way all the tests that use it will have one. Would be good to have a default name created too so that the tests don't need to set names each time unless they need to test specific values